src/umessage/umessagebuilder.rs

```diff
@@ -281,18 +273,16 @@ impl UMessageBuilder {
     ///
     /// ```rust
     /// use up_rust::{UMessageBuilder, UMessageType, UPayloadFormat, UPriority, UUIDBuilder, UUri};
-
     ///
     /// # fn main() -> Result<(), Box<dyn std::error::Error>> {
+    /// # let request_message_id = UUIDBuilder::build();
```

Comment:
the request_message_id is part of the (visible) assertions below so it should not be fenced out FMPOV

Reply:
I'm thinking to make the assertion not visible, since it gives the wrong impression of what a user _should_ be doing.

They should not be plugging a newly created UUID into that slot, but instead using one they extracted from the Request message.

Another way to go about is to fence off creating a Request message using UMessageBuilder, then show how to destructure that to get its reqid.

Kinda like that route, will try.